scripts/generate_sitemap.py

```diff
@@ -102,6 +104,16 @@ def write_sitemap_index(self):
         with open(os.path.join(self.sitemap_dir, 'sitemap_index.xml'), 'w') as f:
             f.write(xml_str)
 
+    def log_errors(self, obj, obj_id, error):
+        if not self.errors:
+            script_utils.add_file_logger(logger, __file__)
+        self.errors += 1
+        logger.info('Error on {}, {}:'.format(obj, obj_id))
+        logger.exception(error)
+        if self.errors == 1000:
+            sentry.log_message('WARNING: generate_sitemap stopped execution after reaching 1000 errors.')
```

Comment:
This should be higher priority than the one where the errors are anywhere from 1-999